src/components/app/ErrorBoundary.js

```diff
@@ -61,35 +68,26 @@ export class ErrorBoundary extends React.Component<Props, State> {
     });
   }
 
-  _toggleErrorDetails = () => {
-    this.setState((state) => ({ showDetails: !state.showDetails }));
-  };
-
   render() {
     if (this.state.hasError) {
-      const { errorString, componentStack, showDetails } = this.state;
+      const { errorString } = this.state;
       return (
         <div className="appErrorBoundary">
           <div className="appErrorBoundaryContents">
             <div className="photon-message-bar photon-message-bar-error photon-message-bar-inner-content appErrorBoundaryMessage">
               <div className="photon-message-bar-inner-text">
                 {this.props.message}
+                {errorString ? ` (${errorString})` : null}.{' '}
+                {this.props.reportExplanationMessage}
               </div>
-              <button
+              <a
                 className="photon-button photon-button-micro photon-message-bar-action-button"
-                type="button"
-                onClick={this._toggleErrorDetails}
-                aria-expanded={showDetails ? 'true' : 'false'}
+                href="https://github.com/firefox-devtools/profiler/issues/new?title=An%20error%20occurred%20in%20Firefox%20Profiler"
```

Comment:
I think we can even populate the content of the bug to include the error message etc. But we can do that later.